init-tools.sh

```diff
@@ -24,7 +24,7 @@ if [ -z "$__DOTNET_PKG" ]; then
     case $OSName in
         Darwin)
             OS=OSX
-            __DOTNET_PKG=dotnet-dev-osx-x64
+            __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-osx-x64
```

Comment:
@joperezr would you mind using the refactoring I have merged in to corefx last week and that you have reviewed? It would be great to have the init-tools.sh as similar as possible in all the repos.

Reply:
will do.